static/templates/message_edit_form.handlebars

```diff
@@ -5,7 +5,7 @@
     <div class="control-group no-margin">
         <label class="control-label edit-control-label" for="message_edit_topic">{{t "Topic" }}</label>
         <div class="controls edit-controls">
-          <input type="text" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" />
+          <input type="text" placeholder="{{t empty_topic }}" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" />
```

Comment:
The {{t ..}} sends the string for translation. In this case we actually don't want to translate it here, but we do where it is first defined, in the javascript (in compose.js). 

Now that I look, it is not actually translated in compose.js -- would it be possible for you to make a separate commit where you add the translation tags to empty_topic_placeholder in compose.js?